spring-boot-project/spring-boot-starters/spring-boot-starter-pulsar-reactive/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+	id "org.springframework.boot.starter"
+}
+
+description = "Starter for using Spring for Apache Pulsar Reactive"
+
+dependencies {
+	api(project(":spring-boot-project:spring-boot-starters:spring-boot-starter"))
+	api("io.projectreactor:reactor-core")
+	api("org.springframework.pulsar:spring-pulsar-reactive")
+}
+
+checkRuntimeClasspathForConflicts {
+	// pulsar bouncycastle ext includes duplicate bouncycastle classes
+	ignore { name -> name.startsWith("org/bouncycastle/") }
+
+	// spring-pulsar also shades/relocates caffeine to the same location
+	ignore { name -> name.startsWith("org/springframework/pulsar/shade/com/github/benmanes/caffeine/") }
+	ignore { name -> name.startsWith("org/springframework/pulsar/shade/com/google/errorprone/") }
+	ignore { name -> name.startsWith("org/springframework/pulsar/shade/org/checkerframework/") }
```

Comment:
Are these still needed? The latest snapshots don't seem to have any shaded classes.

Reply:
We do use the shaded version of the reactive cache https://github.com/spring-projects/spring-pulsar/blob/7f5ced49dc012164037a872933da5b954776437d/spring-pulsar-reactive/spring-pulsar-reactive.gradle#L13

Once I fix this https://github.com/apache/pulsar-client-reactive/issues/127 then we can remove these prefixes and they will not be considered duplicates.

Reply:
https://github.com/spring-projects/spring-boot/pull/37801